com.brave.Brave.metainfo.xml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.brave.Brave</id>
+  <launchable type="desktop-id">com.brave.Brave.desktop</launchable>
+  <name>Brave Browser</name>
+  <developer_name>Brave Software</developer_name>
+  <summary>Community-maintained unofficial Flatpak of Brave Browser</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
```

Comment:
Unless binary is under different license, you don't need extra-data.

Reply:
I'm not sure because of widevine and other proprietary stuff bundled with Brave browser's official binary

Reply:
Yeah, that's a fair point.